app/javascript/src/hyacinth_v1/components/digital_objects/DigitalObjectList.js

```diff
@@ -43,40 +44,87 @@ const storeSearchValues = (
 const DigitalObjectList = (props) => {
   const {
     digitalObjects, displayProjects, displayParentIds, orderBy, totalCount,
-    limit, offset, pageNumber, searchParams, path,
+    limit, offset, pageNumber, searchParams, path, disableDrag,
```

Comment:
Can the drag logic be removed from this component now that you've introduced the other component?

Reply:
Apologies for the confusion of leaving that in. I restored DigitalObjectList.js to the version in main, which appears to include some new formatting adjustments from that branch. DigitalObjectSearch.js has similar formatting adjustments, but I didn't add those - possibly a side effect of the git history cleanup we worked on?

Quick note: there's an update button on the child list page, but the onSubmit is not implemented. I'll add that as part of the HYACINTH-854 work, unless you'd prefer it be ticketed separately. Thanks!

Reply:
@JackBlackLight Are you sure you restored those files from the latest version of main?  I'm not seeing those same changes in https://github.com/cul/ldpd-hyacinth/blob/main/app/javascript/src/hyacinth_v1/components/digital_objects/DigitalObjectList.js

See line 83 in that file, for example.  Your changes will end up reverting lines like that to an earlier state if we merge this in, since your commit will have a newer commit date.  The reason you're seeing the changes as a delta is because they differ from the recently rebased main branch.

Also, I tried your branch locally (to try adding a child object to an Item that didn't have any children yet) and the children page isn't rendering for me.  I'm getting a blank page and a `useState`-related error in the console.

One other note, just to clarify: Remember in my earlier comment when I said this?

> Since not all users will have permission to reorder child objects, this feature should also be implemented on a separate view, meaning that in DigitalObject.js under { routePath: ${path}/children, Component: Children, action: 'read_objects' }, you'll want to add a new route to something like { routePath: ${path}/edit_child_structure, Component: EditChildStructure, action: 'edit_objects' }, (and then create a new EditChildStructure page component).

What I meant was that you didn't need to change the current child view substantially (i.e. you can continue using the DigitalObjectList on the `/children` page) , but that it would be helpful to have the child editor on a separate page located at routePath `${path}/edit_child_structure`.  In this case, you could add a link above the child view that says "Reorder children" and that would bring users to the new page.  One big reason for a new page is that people who have permission to read an object won't always have permission to reorder children.  Adding a new page isn't usually too bad (and you'll see examples of this in `DigitalObject.js` already, along with permission restrictions via `DigitalObjectProtectedRoute`), but let me know if you run into any issues.

Although you could technically allow child reordering on the same page instead of creating a new one, that would involve adding some kind of  `editMode=true` state variable in Children.js and conditionally rendering DigitalObjectList or DigitalObjectChildList based on whether that had been switched on or off by a button -- but I wouldn't recommend it and it would also involve more permission-checking logic and bundling a little too much functionality into Children.js.  Since you already have a working DigitalObjectChildList component, I think you'd be better off with the separate page at this point.

Re: onSubmit button -- Totally fine to keep that non-functional and bundle it into HYACINTH-854 work.  That way we can get this code merged in before HYACINTH-854 is done.